src/style.css

```
@@ -0,0 +1,46 @@
+/* stylelint-disable no-descending-specificity */
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-left: 4px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--0 .stars__star:first-child,
+.stars--1 .stars__star:first-child,
+.stars--2 .stars__star:first-child,
+.stars--3 .stars__star:first-child,
+.stars--4 .stars__star:first-child,
+.stars--5 .stars__star:first-child {
+  margin-left: 0;
+}
```

you can just use `.stars__start:first-child`
```suggestion
.stars__star:first-child {
  margin-left: 0;
}
```